cx_Freeze/finder.py

```diff
@@ -218,7 +218,7 @@ def _FindModule(self, name, path, namespace):
                 # Namespace package (?)
                 module = sys.modules[name]
                 info = ("", "", imp.PKG_DIRECTORY)
-                return None, module.__path__[0], info
+                return None, next(p for p in module.__path__), info
```

Comment:
sigh, I wish that fixed it...

```_NamespacePath``` objects are iterators but NOT generators. In general, my understanding of the python standard-library design philosophy is that they use generators for ephemeral/in-flight data, and iterators for static, iterable data that you might need to save and iterate over more than once. next() only works on fully-qualified generators, and the p for p trick just wraps an iterable object with a brand-new generator.

My solution is certainly a bit clunky, but I couldn't figure out how to get the "first" element off an iterator without flushing the whole iteration. i.e., you could do something like ```list(module.__path__)[0]``` but then you potentially copy the entire list of paths when you only want one, and I figured anything to keep cx_Freeze builds as fast as possible is worth it.

For the record, the code sample you suggested generates the following error instead (I just tested):
```
  return None, next(module.__path__), info
TypeError: '_NamespacePath' object is not an iterator
```

As far as helping you to reproduce it, my setup is pretty vanilla.. windows 10 64-bit, python3.6 installed off the vanilla python installer for windows/amd64, and cx_Freeze installed via pip